include/lib/cpus/aarch64/cortex_a53.h

```diff
@@ -41,6 +41,15 @@
 
 #define CPUECTLR_SMP_BIT		(1 << 6)
 
```

Comment:
It's good practice to define the shift to the retention control bits, even though it is zero in this case.

Also, it would be good to add defines for the Advanced SIMD and Floating-point retention control bits for completeness.